code/datums/config/config_list.dm

```diff
@@ -0,0 +1,138 @@
+	config_description = "BYOND builds that will result the client using them to be banned."
+
+/decl/config/lists/forbidden_versions/get_config_file_value()
+	return jointext(value, " ")
+
+// If the first delay has a custom start time
+// No custom time, no custom time, between 80 to 100 minutes respectively.
+/decl/config/lists/event_first_run
+	uid = "event_first_run"
+	default_value = list(EVENT_LEVEL_MUNDANE = null, EVENT_LEVEL_MODERATE = null,  EVENT_LEVEL_MAJOR = list("lower" = 48000, "upper" = 60000))
+/*
+## The delay until the first time an event of the given severity runs in minutes.
+## Unset setting use the EVENT_DELAY_LOWER and EVENT_DELAY_UPPER values instead.
+# EVENT_CUSTOM_START_MINOR 10;15
+# EVENT_CUSTOM_START_MODERATE 30;45
+EVENT_CUSTOM_START_MAJOR 80;100
+
+	if("event_custom_start_mundane")
+		var/values = text2numlist(value, ";")
+		value[EVENT_LEVEL_MUNDANE] = list("lower" = MinutesToTicks(values[1]), "upper" = MinutesToTicks(values[2]))
+
+	if("event_custom_start_moderate")
+		var/values = text2numlist(value, ";")
+		value[EVENT_LEVEL_MODERATE] = list("lower" = MinutesToTicks(values[1]), "upper" = MinutesToTicks(values[2]))
+
+	if("event_custom_start_major")
+		var/values = text2numlist(value, ";")
+		value[EVENT_LEVEL_MAJOR] = list("lower" = MinutesToTicks(values[1]), "upper" = MinutesToTicks(values[2]))
+*/
+
+// The lowest delay until next event
+// 10, 30, 50 minutes respectively
+/decl/config/lists/event_delay_lower
+	uid = "event_delay_lower"
+	default_value = list(EVENT_LEVEL_MUNDANE = 6000, EVENT_LEVEL_MODERATE = 18000, EVENT_LEVEL_MAJOR = 30000)
+	config_description = list(
+		"The lower delay between events in minutes.",
+		"Affect mundane, moderate, and major events respectively."
+	)
+
+/decl/config/lists/event_delay_lower/set_value(new_value)
+	var/values = text2numlist(new_value, ";")
+	new_value = list(
+		EVENT_LEVEL_MUNDANE  = MinutesToTicks(values[1]),
+		EVENT_LEVEL_MODERATE = MinutesToTicks(values[2]),
+		EVENT_LEVEL_MAJOR    = MinutesToTicks(values[3])
+	)
+	return ..(new_value)
+
+// The upper delay until next event
+// 15, 45, 70 minutes respectively
+/decl/config/lists/event_delay_upper
+	uid = "event_delay_upper"
+	default_value = list(EVENT_LEVEL_MUNDANE = 9000, EVENT_LEVEL_MODERATE = 27000, EVENT_LEVEL_MAJOR = 42000)
+	config_description = list(
+		"The upper delay between events in minutes.",
+		"Affect mundane, moderate, and major events respectively."
+	)
+
+/decl/config/lists/event_delay_upper/set_value(new_value)
+	var/values = text2numlist(new_value, ";")
+	new_value = list(
+		EVENT_LEVEL_MUNDANE  = MinutesToTicks(values[1]),
+		EVENT_LEVEL_MODERATE = MinutesToTicks(values[2]),
+		EVENT_LEVEL_MAJOR    = MinutesToTicks(values[3])
+	)
+	return ..(new_value)
+
+/decl/config/lists/mode_names
+	uid = "mode_names"
+
+// allowed modes
+/decl/config/lists/mode_allowed
+	uid = "modes"
+
+// votable modes
+/decl/config/lists/mode_votable
+	uid = "votable_modes"
```

Comment:
As penny mentioned on discord, ``mode_names``, ``mode_allowed``, ``mode_votable`` aren't really config options, they're just populated and stored in the config options for whatever reasons even if you can't change them via config. There's a few things like that.